### PR TITLE
Add test cases for FhirOperator.kt

### PR DESCRIPTION
**Pull request by RaaziaTarique**
**IMPORTANT: All PRs must be linked to an issue (except for extremely trivial and straightforward changes).**

Fixes https://github.com/google/android-fhir/issues/1075

**Description**
Clear and concise code change description. 

**Alternative(s) considered**
Have you considered any alternatives? And if so, why have you chosen the approach in this PR?

**Type**
Choose one: (Bug fix | Feature | Documentation | Testing | Code health | Builds | Releases | Other)

**Screenshots (if applicable)**

**Checklist**
- [x] I have read and acknowledged the [Code of conduct](https://github.com/google/android-fhir/blob/master/docs/CODE_OF_CONDUCT.md).
- [x] I have read the [Contributing](https://github.com/google/android-fhir/wiki/Contributing) page.
- [x] I have signed the Google [Individual CLA](https://cla.developers.google.com/about/google-individual), or I am covered by my company's [Corporate CLA](https://cla.developers.google.com/about/google-corporate ).
- [x] I have discussed my proposed solution with code owners in the linked issue(s) and we have agreed upon the general approach.
- [x] I have run `./gradlew spotlessApply` and `./gradlew spotlessCheck` to check my code follows the style guide of this project.
- [x] I have run `./gradlew check` and `./gradlew connectedCheck` to test my changes locally.
- [x] I have built and run the demo app(s) to verify my change fixes the issue and/or does not break the demo app(s).

### CODE REVIEW

**Review by ktarasenko — COMMENTED** [review]
Great result!

File: workflow/src/test/java/com/google/android/fhir/workflow/FhirOperatorTest.kt [hunk 1]
```diff
@@ -42,19 +44,25 @@ class FhirOperatorTest {
 
   @Before
   fun setUp() = runBlocking {
-    loadBundle("/ANCIND01-bundle.json")
-    loadBundle("/RuleFilters-1.0.0-bundle.json")
-    loadBundle("/tests-Reportable-bundle.json")
-    loadBundle("/tests-NotReportable-bundle.json")
-
```
[the remark this answers is not in the document]
Comment (RaaziaTarique) [reply workflow/src/test/java/com/google/android/fhir/workflow/FhirOperatorTest.kt:49]: We might need these libraries for some other test cases. I have added the following ones again
`   loadBundle("/tests-Reportable-bundle.json")
    loadBundle("/tests-NotReportable-bundle.json")`

Commented out this one `loadBundle("/RuleFilters-1.0.0-bundle.json")` with `TODO Fix the FHIRHelpers library`  as it's failing the tests.

**Review by ktarasenko — APPROVED** [review]

**Comment by ktarasenko** [comment]
```

com.google.android.fhir.workflow.FhirOperatorTest > evaluateIndividualSubjectMeasure FAILED
    java.lang.OutOfMemoryError at FhirOperatorTest.kt:172
```

Also, enabled automerge

**Comment by RaaziaTarique** [comment]
> ```
> 
> com.google.android.fhir.workflow.FhirOperatorTest > evaluateIndividualSubjectMeasure FAILED
>     java.lang.OutOfMemoryError at FhirOperatorTest.kt:172
> ```
> 
> Also, enabled automerge

These test cases are passing locally and they were passing on CI as well but after merging recent changes from the `master` they are not passing on CI.

**Pull request merged**